user/models.py

```diff
@@ -10,6 +11,24 @@
 from trips.models import UserLocation
 
 
+def change_username(username):
```

Comment:
I'd rather call this `uniquify_username` or similar if it changes the username.
or similar with `slug` if you go for the slug instead.